pyaedt/generic/general_methods.py

```diff
@@ -944,6 +945,74 @@ def is_project_locked(project_path):
     return check_if_path_exists(project_path + ".lock")
 
 
+@pyaedt_function_handler()
+def is_license_feature_available(feature="electronics_desktop", count=1):  # pragma: no cover
+    """Check if license feature is available.
+
+    Parameters
+    ----------
+    feature : str
+        Feature increment name. The default is the electronics desktop one.
+    count : int
+        Number of increments of the same feature available.
+
+    Returns
+    -------
+    bool
+        ``True`` when feature available, ``False`` when feature not available, and
+         ``None`` when licenser server is down.
+    """
+    import subprocess  # nosec B404
+
+    aedt_install_folder = list(installed_versions().values())[0]
+
+    if is_linux:
+        ansysli_util_path = os.path.join(aedt_install_folder, "licensingclient", "linx64", "ansysli_util")
+    else:
+        ansysli_util_path = os.path.join(aedt_install_folder, "licensingclient", "winx64", "ansysli_util")
+    my_env = os.environ.copy()
+
+    tempfile_status = tempfile.NamedTemporaryFile(suffix=".txt", delete=False).name
+    tempfile_checkout = tempfile.NamedTemporaryFile(suffix=".txt", delete=False).name
+
+    # License server status
+    cmd = [ansysli_util_path, "-statli"]
+
+    f = open(tempfile_status, "w")
+
+    subprocess.Popen(cmd, stdout=f, stderr=f, env=my_env).wait()  # nosec
+
+    f.close()
+
+    is_server_down = False
+    with open_file(tempfile_status, "r") as f:
+        for line in f:
+            if line == "ansysli_server process could not be found.\n":
+                is_server_down = True
+                break
+
+    if is_server_down:
+        pyaedt_logger.info("License server process could not be found.")
+        return None
```

Comment:
```suggestion
        pyaedt_logger.info("License server process could not be found.")
        return None
```
Just wondering, would it make sense to log a warning here and return `False` instead of the current code ?
I kinda feel like this function will be used in if statement like  `if is_license_feature_available(): ...` thus I'm not sure if anyone will differentiate between False and None.

Reply:
In fact I did it in purpose just in case the user needs to check if the server is alive or not. But I agree. I will modify it